examples/console/index.css

```diff
@@ -19,6 +19,12 @@ body {
   bottom: 4px;
 }
 
+
+.p-CommandPalette {
```

Comment:
This is just the console example application's command palette, so it's in the style sheet for that example.